megamek/src/megamek/client/bot/princess/WeaponFireInfo.java

```diff
@@ -501,154 +496,164 @@ void initDamage(@Nullable final MovePath shooterPath,
                     final boolean assumeUnderFlightPath,
                     final boolean guess,
                     final int[] bombPayload) {
+        boolean debugging = false;
+        
         final StringBuilder msg =
-                new StringBuilder("Initializing Damage for ").append(getShooter().getDisplayName())
+                debugging ?
+                        new StringBuilder("Initializing Damage for ").append(getShooter().getDisplayName())
                                                              .append(" firing ").append(getWeapon().getDesc())
                                                              .append(" at ").append(getTarget().getDisplayName())
-                                                             .append(":");
+                                                             .append(":") :
+                        new StringBuilder();
```

Comment:
I'd make this `null` to avoid the default allocation size of the underlying `char[]`.

Reply:
Ok.